src/cli/init.zig

```diff
@@ -0,0 +1,312 @@
+const std = @import("std");
+const args = @import("args");
+const jetzig_install = @import("jetzig_install");
+
+pub const Options = struct {
+    path: ?[]const u8 = null,
+
+    pub const shorthands = .{
+        .p = "path",
+    };
+
+    pub const meta = .{
+        .usage_summary = "[--path PATH]",
+        .full_text =
+        \\Initializes a new Jetzig project in the current directory or attempts to
+        \\create a new directory specified by PATH
+        \\
+        \\Creates build.zig, build.zig.zon, src/main.zig, and an example view with a template.
+        \\
+        \\Run `zig build run` to launch a development server when complete.
+        ,
+        .option_docs = .{
+            .path = "Set the output path relative to the current directory (default: current directory)",
+        },
+    };
+};
+
+/// Run the `jetzig init` command.
+pub fn run(
+    allocator: std.mem.Allocator,
+    options: Options,
+    writer: anytype,
+    other_options: struct { help: bool },
+) !void {
+    if (other_options.help) {
+        try args.printHelp(Options, "jetzig init", writer);
+        return;
+    }
+
+    var install_dir = if (options.path) |path| try std.fs.cwd().makeOpenPath(path, .{}) else std.fs.cwd();
+
+    const real_path = try install_dir.realpathAlloc(allocator, ".");
+    defer allocator.free(real_path);
+
+    const output = try std.fmt.allocPrint(allocator, "Creating new project in {s}\n\n", .{real_path});
+    defer allocator.free(output);
+    try writer.writeAll(output);
+
+    const project_name = "test";
+
+    try copySourceFile(
+        allocator,
+        install_dir,
+        "demo/build.zig",
+        "build.zig",
+        &[_]Replace{.{ .from = "jetzig-demo", .to = project_name }},
+    );
+
+    try copySourceFile(
```

Comment:
The demo build.zig.zon is not necessary, the fetch will generate it for you

Reply:
Nice - thanks.